cli/impl/src/main/java/io/helidon/build/cli/impl/Metadata.java

```diff
@@ -150,35 +154,35 @@ public FileTime lastUpdateTime() {
     }
 
     /**
-     * Returns the latest Helidon version.
+     * Returns information about archetypes versions.
      *
-     * @return The version.
+     * @return ArchetypesData
      * @throws UpdateFailed if the metadata update failed
      */
-    public MavenVersion latestVersion() throws UpdateFailed {
-        return latestVersion(false);
+    ArchetypesData archetypesData() throws UpdateFailed  {
```

Comment:
Instead of adding a new method and updating the usage of `Metadata.latest()` ; can you directly update `Metadata.latest()` ?